docker-compose.yml

```
@@ -1,10 +1,16 @@
 services:
   db:
     image: postgres:13.14
+    environment:
+      POSTGRES_PASSWORD: devpassword
     ports:
       - "5432:5432"
+  timescale:
+    image: timescale/timescaledb-ha:pg16.2-ts2.14.2-all
```

This image is 1.5GB @jonatas, isn't there any small one suitable for development + CI?

---

Yes, there's a smaller one but does not contains the toolkit. Probably [latest-pg16](https://hub.docker.com/layers/timescale/timescaledb/latest-pg16).
```suggestion
    image: timescale/timescaledb:latest-pg16
```

---

Link 404s, https://hub.docker.com/layers/timescale/timescaledb/latest-pg16. What's the toolkit useful for?

---

Toolkit can give extra flexibility in the components for statistics, containing several cool stuff like [most of the hyperfunctions](https://docs.timescale.com/api/latest/hyperfunctions/).

It allow us to use components like stats_agg that have some embeddings for several statistics, so you can extract stddev, average without recomputing everything. Percentile_agg do the same for distribution and allow you to get several discs without reprocessing. The coolest part is rollup that allows to merge this _agg objects and reuse pre-calculated data into bigger timeframes.

https://hub.docker.com/layers/timescale/timescaledb/latest-pg16/images/sha256-379dc1846318a812301a78e860f7af5f09fd391f8de798a4b477827174da8f96?context=explore

---

I'm investigating if we have any light image that includes the toolkit, but we can also move without it and later just add it as a plus.

I updated it now and tested again 👍